src/app/dynamic-forms/fields/link/link.component.html

```diff
@@ -143,8 +143,7 @@
       </mat-autocomplete>
     </ng-container>
     <ng-container *ngIf="!canEdit && !downloadLink">
-      <label class="pre-formatted" *ngIf="showActualField">
-        {{ selectedValue?.label || "-" }}
+      <label class="pre-formatted" *ngIf="showActualField" [innerHTML]="selectedValue && selectedValue.label ? selectedValue.label : '-'">
```

Comment:
You can use optional chaining here to make the template cleaner.

`[innerHTML]="selectdValue?.label || '-'"`

Reply:
done